content/blog/2020-04-16-april-20-community-gems.md

```diff
@@ -0,0 +1,151 @@
+---
+title: April '20 Community Gems
+date: 2020-04-16
+description: |
+  Great discussions and technical Q&A's from our users.
```

Comment:
this is what is visible on the front page - use the same as longDescription?

Reply:
done